doc/nvim-treesitter.txt

```diff
@@ -170,7 +170,9 @@ Note: This is highly experimental, and folding can break on some types of
 ==============================================================================
 HIGHLIGHTS                                        *nvim-treesitter-highlights*
 
-TSError
+`TSError`
+							          *hl-TSError*
+For syntax/parser errors.
```

Comment:
is this highlight group disabled by default ? If so, you could write about that here.

Reply:
It's enabled. I would prefer disabled.

Reply:
Then we can easily disable by default, can you change this in this PR ?

Reply:
You can change it to default=Off if you like.